src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -144,6 +144,65 @@ protected virtual JObject InvokeScript(JArray _params)
             return GetInvokeResult(script, signers);
         }
 
+        [RpcMethod]
+        protected virtual JObject InvokeIterator(JArray _params)
```

Comment:
Can someone please provide the reasoning behind adding this new Rpc method? What's the use case for it?

Reply:
> Can someone please provide the reasoning behind adding this new Rpc method? What's the use case for it?

You could find `Iterator` in (Nep11 NFT Token)[https://github.com/neo-project/proposals/blob/nft/nep-11.mediawiki] and it's really important so that we need this RPC.

Reply:
Iterate if the result it's an interator automatically?

Reply:
> Iterate if the result it's an interator automatically?

But it takes two additional parameters, startIndex and endIndex,

Reply:
> Iterate if the result it's an interator automatically?

Then why we use list? It's using iterator for a senario that if huge amount of data, we could get it one by one so that not cause any response issue.